### PR TITLE
Move the documentation from Wiki to GitHub

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just a best-effort migration.
The syntax on https://wiki.jenkins.io/display/JENKINS/Parameterized+Trigger+Plugin is pretty complicated, and it looks like some information entries got lost.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,126 @@
...
+-   **Subversion revision**: makes sure the triggered projects are built with the same revision(s) of the triggering build.
+    You still have to make sure those projects are actually configured to checkout the right Subversion URLs.
+-   **Restrict matrix execution to a subset**: allows you to specify the same combination filter expression as you use in the matrix project configuration and further restricts the subset of the downstream matrix builds to be run.
+
+The parameter section is itself pluggable, and other plugins can contribute other sources of parameters.
+
+This triggering mechanism can be used both as a post-build step or as a build step, in which case you can also block for the completion of the triggered builds.
+This lets you create a "function call" like semantics.
+
+**\*\*\* YOU MUST DEFINE THE PARAMETER IN DOWNSTREAM JOBS VIA  "This
```
Comment (zbynek) [inline README.md]: Would be cool to avoid the asterisks, maybe ⚠️ ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+    without Item.READ (JENKINS-31727)
+-   Minor fixes and improvements ([PR
+    \#91](https://github.com/jenkinsci/parameterized-trigger-plugin/pull/91))
+
+#### 2.29
+
+Release date: (Sep 10, 2015)
+
+-   Workflow compatibility fixes
+-   Extra unit tests for JENKINS-30040
```
Comment (zbynek) [inline CHANGELOG.md]: the Jenkins references could be replaced by links

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+-   Prevent NullPointerException when Parameterized Trigger starts
+    multiple builds and rejects some of them due to the missing
+    Job/Build permission
+    ([JENKINS-45471](https://issues.jenkins-ci.org/browse/JENKINS-45471))
+
+#### 2.35
+
+Release date: (July 10, 2017)
+
+-   [Fix security
```
Comment (zbynek) [inline CHANGELOG.md]: Strange linebreak

Comment (oleg-nenashev) [reply CHANGELOG.md]: Pandoc did it. Did not want to spend too much time on polishing the changelog

Comment (timja) [reply CHANGELOG.md]: it renders fine 👍

**Comment by oleg-nenashev** [comment]
The image was quit obsolete, and it was not really used in the text.
I can capture a new screenshot

File: CHANGELOG.md (hunk 2 above)
Comment (oleg-nenashev) [reply CHANGELOG.md]: Yes, just too much work for a historical changelog. It was as is on the Wiki

**Comment by oleg-nenashev** [comment]
Will follow-up with images in follow-up once the pending PRs are merged

**Pull request merged**